src/de/metas/jenkins/DockerConf.groovy

```diff
@@ -1,10 +1,10 @@
 package de.metas.jenkins
 
 class DockerConf implements Serializable {
-    public static final String PUSH_REGISTRY = 'nexus.metasfresh.com:6001'
+    public static final String PUSH_REGISTRY = 'dockertemp.metasfresh.com'
```

Comment:
change hostname to docker.metasfresh.com before integrating; don't re-add the port name on julian's advise